### DISCUSSION TITLE
Obtaining 3D Information for PCQM4M-LSC from PubChemQC

### DISCUSSION THREAD

**Discussion by Prometheus1400** [snap-stanford/ogb#243] [PCQM4M-LSC]
Our team is interested in obtaining the 3D information from the original PubChemQC dataset for the molecules in the PCQM4M dataset for research purposes. However it looks like the majority of the molecules (smiles strings) in the PCQM4M dataset are not present in the PubChemQC dataset we downloaded.

Do you have any documentation or code for how you formed the PCQM4M dataset from the PubChemQC dataset?

**Comment by weihua916** [comment]
Hi! Thanks for the inquiry. 

We agree that 3D information is important, and we are likely to release DFT-calculated 3D structure for training molecules in the next month or so. Stay tuned!

**Comment by linminhtoo** [comment]
hi, I have a related question. is there any publication detailing how the `PCQM4M` dataset was constructed from the `PubChemQC` dataset? or any code for it?

**Reply by weihua916** [reply]
Hi! There is no documentation, but we simply extracted molecules and HOMO-LUMO gap from the PubChemQC database. What information exactly do you want?

**Reply by KuzmaKhrabrov** [reply]
> Hi! There is no documentation, but we simply extracted molecules and HOMO-LUMO gap from the PubChemQC database. What information exactly do you want?

Could you, please, tell, 
1. which "SMILES" strings were taken? 
2. which variant of the database was used (json-files or postgres database)?
3. How were the rdkit Mol objects generated?
4. Why are there 368015 samples without 3D geometries?
